bin/install

```diff
@@ -50,16 +48,26 @@ install_default_python_packages() {
   fi
 }
 
-
 fetch_all_versions() {
-  $(python_build_path) --definitions | grep -E '^\d+\.\d+\.\d+$' | sort -rV
+  $(python_build_path) --definitions
+}
+
+reverse_sort_numeric_versions() {
+  egrep '^([0-9]+\.){2}[0-9]+$' | LC_ALL=C sort -t. -nrk1 -nrk2 -nrk3
```

Comment:
shellcheck complains about using `egrep` as it's deprecated

Reply:
Oh, okay.